Content/Application/ContentResolver/Value/ContentView.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\ContentBundle\Content\Application\ContentResolver\Value;
+
+/**
+ * @internal This class is intended for internal use only within the package/library. Modifying or depending on this class may result in unexpected behavior and is not supported.
+ */
+class ContentView
+{
+    /**
+     * @param mixed[] $view
+     */
+    private function __construct(
+        private mixed $content,
+        private array $view
+    ) {
+    }
+
+    /**
+     * @param mixed[] $view
+     */
+    public static function create(mixed $content, array $view): self
+    {
+        return new self($content, $view);
+    }
+
+    /**
+     * @param mixed[] $view
+     */
+    public static function createResolvable(string|int $id, string $resourceLoaderKey, array $view): self
+    {
+        $resolvableResources = [
+            new ResolvableResource($id, $resourceLoaderKey),
+        ];
+
+        return new self($resolvableResources, $view);
```

Comment:
```suggestion
        return new self(new ResolvableResource($id, $resourceLoaderKey), $view);
```

This returns an array but should not return an array as `content.media` with a single selection should directly be returned not as an array.